banner/tests/requirements/requirements.txt

```diff
@@ -0,0 +1,4 @@
+django-contrib-comments==1.7.1
+psycopg2
+six==1.9.0
+django-ultracache==1.10.2
```

Comment:
I'm not sure ultracache should be a requirement on a standalone product.

Reply:
Whoops, this sneaked in when I was restructuring the module

Reply:
Reminder to remove it please.